apps/docs/src/content/reference/xr/getting-started.mdx

```diff
@@ -29,11 +30,18 @@ The following adds a button to start your session and controllers inside an XR m
   import Scene from './scene.svelte'
 </script>
 
-<Canvas>
-  <Scene />
-</Canvas>
-
-<VRButton />
+<div>
+  <Canvas>
+    <Scene />
+  </Canvas>
+  <VRButton />
+</div>
+
+<style>
```

Comment:
Can we remove the div / styling in this case since this is just a snippet?

Reply:
sure thing 👍